data-plane/pom.xml

```diff
@@ -59,8 +59,8 @@
     <fabric8.kubernetes.version>6.13.4</fabric8.kubernetes.version>
     <kafka.version>3.7.1</kafka.version>
     <debezium.version>2.6.1.Final</debezium.version>
-    <jib.version>3.4.4</jib.version>
-    <quarkus.version>3.15.3.SP1-redhat-00002</quarkus.version>
+    <jib.version>3.4.6</jib.version>
+    <quarkus.version>3.15.6.redhat-00002</quarkus.version>
```

Comment:
did this job also run `mvn clean ...` to update the `THIRDPARTY.txt` file?

Reply:
Actually we have a [workflow](https://github.com/openshift-knative/eventing-kafka-broker/blob/release-v1.16/.github/workflows/dependabot-deps.yaml) which runs `make generate-release` on PRs from dependabot. But this seems to not run correctly ATM :(

Reply:
Ah wait. Not sure if `make generate-release` runs `mvn clean ...`